src/Tribe/Assets.php

```diff
@@ -705,7 +710,7 @@ public function register( $origin, $slug, $file, $deps = [], $action = null, $ar
 
 		// Clean these
 		$asset->priority  = absint( $asset->priority );
-		$asset->in_footer = (bool) $asset->in_footer;
+		$asset->in_footer = $asset->in_footer; // Since WordPress 6.3, this parameter accepts an array argument.
```

Comment:
```suggestion
		$asset->in_footer = is_array($asset->in_footer) ? $asset->in_footer : (bool)$asset->in_footer; // Since WordPress 6.3, this parameter accepts an array argument.
```
For better compatibility, we probably should check if array else cast to bool.

Reply:
Makes sense, thanks!